packages/protocol/contracts-0.8/stability/FeeCurrencyAdapter.sol

```diff
@@ -162,7 +162,7 @@ contract FeeCurrencyAdapter is Initializable, CalledByVm, IFeeCurrencyAdapter {
   }
 
   function downscale(uint256 value) internal view returns (uint256) {
```

Comment:
Can't comment on that line, but debit should explicitly have in the natspec that it is theoretically possible to get more debited than the user specified.

Reply:
I updated comment and added example as discussed with @mcortesi